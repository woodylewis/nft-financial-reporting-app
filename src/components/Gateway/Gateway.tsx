import * as React from 'react';
import { BrowserRouter, Routes, Route } from 'react-router-dom';

import TopBar from '../TopBar/TopBar';
import Home from '../Home/Home';
import Network from '../Network/Network';
import { StringList } from '../../constants/StringList';

const Gateway = () => {
  return (
    <div>
      <BrowserRouter>
        <TopBar />
        <Routes>
          <Route path='/' element={<Home />} />
          <Route index element={<Home />} />
          <Route path={StringList.NETWORK_LINK} element={<Network />} />
        </Routes>
      </BrowserRouter>
    </div>
  );
};

export default Gateway;
