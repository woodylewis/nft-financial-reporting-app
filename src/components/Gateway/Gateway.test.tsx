import * as React from 'react';
import { render, cleanup } from '@testing-library/react';
import Gateway from './Gateway';

describe('Gateway component', () => {
  afterEach(cleanup);

  it('should match snapshot', () => {
    const { asFragment } = render(<Gateway />);
    expect(asFragment).toMatchSnapshot();
  });
});
