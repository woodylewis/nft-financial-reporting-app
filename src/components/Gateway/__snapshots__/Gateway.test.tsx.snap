// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gateway component should match snapshot 1`] = `[Function]`;
